### DISCUSSION TITLE
Search bar for guest actions/thoughts

### DISCUSSION THREAD

**Discussion by astraylife1** [OpenRCT2/OpenRCT2#19476] [Feature requests]
It would be great if there was a search option under guest thoughts and actions. It’s frustrating having to scroll through to find how many guests are “heading for x ride” for example. If we could just start typing “heading” and it will sort them out that would be awesome.

**Comment by sjoerddebruin** [comment]
A button in the ride window for "heading to" would be nice as well imo.
